### PR TITLE
Added NFT Marketplace guided project

### PR DESCRIPTION
**Pull request by navyansh007**
As per the feedback, I converted this project into a guided project, which will guide learners to make an NFT Marketplace- a real-world project that will help them to learn key Web3 and Blockchain concepts. By building this project, they will also get the experience of making something that can be used by actual users.

I, as the developer of this starter code and guided project, hope that this will add value to the learning journey of the budding blockchain developers. I am open to corrections that can be done and would be more than happy if this project gets added to CodeDamn's list of various interesting projects.

### CODE REVIEW

File: nft-marketplace/step2/challenges.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name: "Hosting NFTs and Metadata to IPFS"
+stepId: step2
```
Comment (Pranavtechie) [inline nft-marketplace/step2/challenges.yml:2]: Can you add the challenges for the step2

**Review by Pranavtechie — CHANGES_REQUESTED** [review]

File: nft-marketplace/step2/challenges.yml (hunk 1 above)
Comment (navyansh007) [reply nft-marketplace/step2/challenges.yml:2]: There is just one step here which is very basic, so the step breakdown is still required?

Comment (navyansh007) [reply nft-marketplace/step2/challenges.yml]: I have added the possible step breakdown in step 2. Please review the change, and if everything is good, please merge the branch.

**Comment by navyansh007** [comment]
Hey @Pranavtechie
Please add hacktoberfest label to the repo so that this contribution gets eligible for Hacktoberfest.

**Comment by navyansh007** [comment]
Hello, @Pranavtechie @mehulmpt @rishabhrao
Just a kind reminder, the changes requested by @Pranavtechie and @mehulmpt have been implemented in the code. Please merge the code in the master branch of Codedamn's projects repo if everything is good to go.

**Comment by navyansh007** [comment]
Hey @Pranavtechie , 
Since this PR has been accepted, may you please tell me what are the next steps?

**Comment by Pranavtechie** [comment]
@navyansh007 please share your email address here , or reach out to me at pranav@codedamn.com

**Comment by navyansh007** [comment]
Here is my email id: navyanshkesarwani@gmail.com

**Pull request merged**